usaspending_api/references/tests/integration/test_load_psc.py

```diff
@@ -1,10 +1,15 @@
-import pytest
+import logging
+from pathlib import Path
+from datetime import datetime
 
+import pytest
 from django.core.management import call_command
-from datetime import datetime
-from usaspending_api.references.models import PSC
 from model_bakery import baker
 
+from usaspending_api.references.models import PSC
+
+logger = logging.getLogger(__name__)
```

Comment:
Any reason to keep this logger defined in the test?

Reply:
Good catch.